site/en/docs/extensions/reference/declarativeNetRequest/index.md

```diff
@@ -9,19 +9,22 @@ has_warning: One or more of these permissions <a href="/docs/extensions/mv3/perm
 
 ## Manifest {: #manifest }
 
-Extensions must declare either the `declarativeNetRequest` or the
-`declarativeNetRequestWithHostAccess` (available since **Chrome 96**) permission in the extension
-[manifest][1] to use this API. The former allows extensions to block and upgrade requests without
-any [host permissions][2]. Host permissions are still required if the extension wants to redirect a
-request or modify headers on it. The `declarativeNetRequestWithHostAccess` permission always
-requires host permissions to the request URL and initiator to act on a request.
+### Permissions {: #permission-definitions }
 
-The `declarativeNetRequestFeedback` permission is required to access functions and events which
-return information on declarative rules matched.
+`declarativeNetRequest`
+: Allows extensions to block and upgrade requests.
 
-To specify static [Rulesets][3], extensions must also declare the `"declarative_net_request"`
-manifest key, which should be a dictionary with a single key called `"rule_resources"`. It should be
-a list containing dictionaries of type [Ruleset][3], as shown below.
+`declarativeNetRequestWithHostAccess`
+: Allows extensions to redirect requests and modify request and response headers. This permission always requires [host permissions](/docs/extensions/mv3/declare_permissions/) to the requestED URL and initiator to act on a request.
+
+`declarativeNetRequestFeedback`
+: Allows access to functions and events that return information on matched rules during debugging. This permission only works in [unpacked extensions](/docs/extensions/mv3/getstarted/development-basics/#load-unpacked).
+
+Certain rules such as redirect and header modification rules also require host permissions that match the network requests that these rules apply to.
+
+### Static rulesets {: #manifest-rule-resources }
```

Comment:
It feels like this should come after the concepts section where we explain what a ruleset is?